nuxt.config.js

```diff
@@ -144,6 +163,7 @@ module.exports = {
   },
 
   plugins: [
-    { src: `~plugins/vimeo-player`, ssr: false }
+    { src: `~plugins/vimeo-player`, ssr: false },
+    { src: '~/plugins/install-notification.js', ssr: false }
```

Comment:
`install-prompt` as it's a prompt rather than a notification
(and a trailing `,` please)